src/smart-components/role/role-permissions.scss

```diff
@@ -5,4 +5,21 @@
       font-weight: lighter;
     }
   }
+
+  .ins-c-text__disabled {
+    color: var(--pf-global--Color--200);
+  }
+}
+
+.ins-m-resource-definitions {
+  .pf-c-modal-box__body {
```

Comment:
If you can find PF variables with same values that would be cool :) If not that is also fine

Reply:
replaced existing ones, thanks @PanSpagetka